pulp_2to3_migration/app/pre_migration.py

```diff
@@ -480,3 +480,14 @@
                     pub_model.objects.all().delete()
                 for distribution_model in distributor_migrator.pulp3_distribution_models:
                     distribution_model.objects.all().delete()
+            for pulp2_repo in Pulp2Repository.objects.filter(type=plugin.type, is_migrated=True):
```

Comment:
Is there any reason to not simply wipe out the premigrated repos as well?  Would simplify this code.

Reply:
Apart from content units, repositories are the only thing we don't currently wipe out every time.